src/modules/profile/pages/profileSettings/components/CreatorNotifications.tsx

```diff
@@ -0,0 +1,180 @@
+        prevSettings.map((setting) =>
+          setting.notificationType === type
+            ? {
+                ...setting,
+                configurations: setting.configurations.map((config) =>
+                  config.name === name ? { ...config, value } : config,
+                ),
+              }
+            : setting,
+        ),
+      )
+    } catch (error) {
+      console.error('Failed to update notification setting:', error)
+    }
+  }
+
+  if (!project) return null
+
+  return (
+    <VStack spacing={6} align="stretch" width="100%">
+      <Body size="lg" bold>
+        Creator notifications
+      </Body>
+      <VStack spacing={4} align="stretch" p={4}>
+        <HStack>
+          <Avatar src={project.avatarUrl} name={project.title} size="sm" />
+          <Body bold>{project.title}</Body>
+        </HStack>
+
+        <FormControl display="flex" alignItems="center" justifyContent="space-between">
+          <FormLabel htmlFor="creator-summary" mb="0">
+            <Body size="lg">Creator summary email</Body>
+          </FormLabel>
+          <HStack spacing={4}>
+            <Select
+              value={getConfigValue('creator.projectSummary', 'frequency') || ''}
+              onChange={(e) => updateConfigValue('creator.projectSummary', 'frequency', e.target.value)}
+              size="sm"
+              placeholder="Select frequency"
+              width="auto"
+            >
+              <option value="weekly">Weekly</option>
+              <option value="monthly">Monthly</option>
+            </Select>
+            <Switch
+              id="creator-summary"
+              isChecked={getConfigValue('creator.projectSummary', 'is_enabled') === 'true'}
+              onChange={(e) =>
+                updateConfigValue('creator.projectSummary', 'is_enabled', e.target.checked ? 'true' : 'false')
+              }
+            />
+          </HStack>
+        </FormControl>
+        <Body size="sm">
+          Receive a monthly email about your project summary: stats, goal progress and, hot rewards..
+        </Body>
+
+        <Body size="lg" bold>
+          Event alerts
+        </Body>
+        <Body size="sm">
+          Get notified with an email as soon as important events takes place in your project, from making a reward sale,
+          to receiving a contribution above a certain amount, and reaching a goal.
+        </Body>
+
+        <FormControl display="flex" alignItems="center" justifyContent="space-between">
+          <FormLabel htmlFor="goal-reached" mb="0">
+            <Body size="lg">Goal Reached Email</Body>
+          </FormLabel>
+          <Switch
+            id="goal-reached"
+            isChecked={getConfigValue('creator.goalReached', 'is_enabled') === 'true'}
+            onChange={(e) =>
+              updateConfigValue('creator.goalReached', 'is_enabled', e.target.checked ? 'true' : 'false')
+            }
+          />
+        </FormControl>
```

Comment:
This, FormControl, FormLabel & <Input>, seems to be quite repeated here, would make sense to create a wrapper component with, formControl, formLabel, that takes `children`, `label` & `htmlfor` as input params. The rest is reusable.

We already have something like this, we use called the FormField but I think that is vertial, This could be a HorizontalFormField.